Makefile

```diff
@@ -0,0 +1,34 @@
+.PHONY: test deps build56
```

Comment:
Needs `test-all`, `test-all-7` etc?

Reply:
Not sure to be honest, my first time writing a Makefile :)